React/Lesson40/communication-react-app/src/Product/Product.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './Product.css';
+
+const Product = ({ product, onAddToCart }) => {
+  const handleAddToCartClick = () => {
+    console.log('Product add to Cart:', product);
+    onAddToCart(product);
```

Comment:
since you know already about context you have to use it and to pass function as a param